site/docs/sql/intro.md

```diff
@@ -1,351 +1,399 @@
+**Note:** _The service is in beta state and supports only a very limited
+subset of the planned functionality. The behavior, API, and binary
+formats will probably change in future releases._
 
-## Overview
+## Quick Start
 
-In the first release, Jet SQL supports the following features:
+Prerequisite is Java.
 
-- SQL queries over [Apache Kafka topics](kafka-connector.md) and
-[files (local and remote)](file-connector.md)
-- Joining Kafka or file data with local IMaps (enrichment)
-- Filtering and projection using [SQL
-expressions](https://docs.hazelcast.org/docs/{imdg-version}/manual/html-single/index.html#expressions)
-- Aggregating data from files using predefined [aggregate
-functions](basic-commands#aggregation-functions)
-- Receiving query results via Jet client (Java) or writing the results
-to an [IMap](imap-connector.md) in the Jet cluster
-- Running continuous (streaming) and batch queries, see
-[Job Management](job-management.md)
-
-These are some of the features on our roadmap:
-
-- Joins with arbitrary external data sources
-- Windowed aggregation of streaming data
-- JDBC
-
-## Installation
+Download and start Jet by pasting this into your terminal:
 
-When you use the distribution package, the `hazelcast-jet-sql` is on
-the class path by default. Nevertheless, make sure to have the
-`hazelcast-jet-sql-{jet-version}.jar` file in the `lib/`
-directory.
-
-If you use Jet in embedded mode, besides the `hazelcast-jet` dependency
-add also the `hazelcast-jet-sql` dep:
-
-<!--DOCUSAURUS_CODE_TABS-->
-
-<!--Gradle-->
-
-```groovy
-compile 'com.hazelcast.jet:hazelcast-jet-sql:{jet-version}'
+```text
+wget https://github.com/hazelcast/hazelcast-jet/releases/download/v4.4/hazelcast-jet-4.4.tar.gz
+tar xvf hazelcast-jet-4.4.tar.gz
+cd hazelcast-jet-4.4
+bin/jet-start
 ```
 
-<!--Maven-->
+You are now ready to write some Jet SQL. Start another terminal window
+and try this:
 
-```xml
-<dependency>
-    <groupId>com.hazelcast.jet</groupId>
-    <artifactId>hazelcast-jet-sql</artifactId>
-    <version>{jet-version}</version>
-</dependency>
+```sql
+$ bin/jet sql
+Connected to Hazelcast Jet 4.4 at [192.168.5.13]:5701 (+0 more)
+Type 'help' for instructions
+sql> select * from TABLE(generate_series(1,5));
++------------+
+|           v|
++------------+
+|           1|
+|           2|
+|           3|
+|           4|
+|           5|
++------------+
+5 row(s) selected
+sql> select sum(v) from TABLE(generate_series(1,5));
++--------------------+
+|              EXPR$0|
++--------------------+
+|                  15|
++--------------------+
+1 row(s) selected
+sql>
 ```
 
-<!--END_DOCUSAURUS_CODE_TABS-->
+Here are two more examples with streaming SQL. Streaming queries never
+complete, so use `Ctrl-C` to cancel them after a while:
 
-If you use other features from SQL such as the Kafka or File connectors,
-also add the `hazelcast-jet-kafka` or `hazelcast-jet-hadoop` modules in
-the same way.
+```sql
+sql> SELECT * FROM TABLE(generate_stream(10));
```

Comment:
There should be a note what `generate_stream` does - the argument isn't clear. Or do this:
```suggestion
sql> SELECT * FROM TABLE(generate_stream(rate => 10));
```
But that also deserves a note because named arguments aren't that well known.